src/dosbox.cpp

```diff
@@ -597,19 +597,26 @@ void DOSBOX_Init(void) {
 	Pbool = secprop->Add_bool("nosound",Property::Changeable::OnlyAtStart,false);
 	Pbool->Set_help("Enable silent mode, sound is still emulated though.");
 
-	Pint = secprop->Add_int("rate",Property::Changeable::OnlyAtStart,44100);
+	Pint = secprop->Add_int("rate", only_at_start, 48000);
 	Pint->Set_values(rates);
 	Pint->Set_help("Mixer sample rate, setting any device's rate higher than this will probably lower their sound quality.");
 
-	const char *blocksizes[] = {
-		 "1024", "2048", "4096", "8192", "512", "256", 0};
-	Pint = secprop->Add_int("blocksize",Property::Changeable::OnlyAtStart,1024);
+	const char *blocksizes[] = {"1024", "2048", "4096", "8192",
+	                            "512",  "256",  0};
+	Pint = secprop->Add_int("blocksize", deprecated, 1024);
 	Pint->Set_values(blocksizes);
-	Pint->Set_help("Mixer block size, larger blocks might help sound stuttering but sound will also be more lagged.");
+	Pint->Set_help("This property is deprecated, use latency instead.");
 
-	Pint = secprop->Add_int("prebuffer",Property::Changeable::OnlyAtStart,25);
+	Pint = secprop->Add_int("prebuffer", deprecated, 25);
 	Pint->SetMinMax(0,100);
-	Pint->Set_help("How many milliseconds of data to keep on top of the blocksize.");
+	Pint->Set_help("This property is deprecated, use latency instead.");
+
+	Pint = secprop->Add_int("latency", only_at_start, 15);
+	Pint->SetMinMax(1, 100);
+	Pint->Set_help("Desired audio latency in milliseconds. Range is 1-100.");
+
+	Pbool = secprop->Add_bool("negotiate", only_at_start, true);
+	Pbool->Set_help("Allow DOSBox to negotiate preferred rate and latency with the system audio driver.");
```

Comment:
I think it's the other way 'round: negotiation let's the system possibly override the user's settings (negotiation-true means OS/drivers trump).  When negotiation is false, the user setting trumps.

I think the way it's written, if we take the false scenario, "don't allow DOSBox to negotiate the preferred rate and latency with the system audio driver." (In other words, the system audio driver is in charge). 

Or maybe I have it backwar

Reply:
how about

```
	Pbool->Set_help("Allow system audio driver to negotiate optimal rate and latency values.");
```

Reply:
Read perfect; now matches exactly with what I'm seeing.

Reply:
Is this clear enough that it's not an `auto` type setting? It will still start with the indicated `rate` and `latency` and try to get close, but it's not something where SDL can just start from zero.